cmd/harmony/main.go

```diff
@@ -178,6 +178,10 @@ func getHarmonyConfig(cmd *cobra.Command) (harmonyConfig, error) {
 		return harmonyConfig{}, err
 	}
 
+	if config.General.IsOffline {
```

Comment:
From my point of view, this piece of code are not supposed to be here. Can we move this piece of code to `applyGeneralFlags`, and then in `validateHarmonyConfig`, return an error when the p2p IP and offline settings is not consistent?